### PR TITLE
Add Color Conversions, Allow Changing Display Background Color

### PR DESCRIPTION
**Pull request by crzysdrs**
Adds some useful color conversions implementations so it's easier to work with the `image` and `embedded-graphics` crate.

Allow setting display background color (i.e. the border around the display), which I only recently realized was an option.

### CODE REVIEW

File: src/color.rs [hunk 1]
```diff
@@ -72,9 +72,70 @@ impl From<()> for OctColor {
...
+    fn from(b: embedded_graphics::pixelcolor::raw::RawU4) -> Self {
+        use embedded_graphics::prelude::RawData;
+        OctColor::from_nibble(b.into_inner()).unwrap()
+    }
+}
+
 #[cfg(feature = "graphics")]
 impl PixelColor for OctColor {
-    type Raw = ();
+    type Raw = embedded_graphics::pixelcolor::raw::RawU4;
```
Comment (caemor) [inline src/color.rs:138]: Did this change with the api or is this more efficient?

**Comment by caemor** [comment]
Thanks for the PR!
Could you please switch to embedded_graphics_core (see #85)?
Everything else looks good and ready to merge to me.

File: src/color.rs (hunk 1 above)
Comment (crzysdrs) [reply src/color.rs]: It's actually just more correct, unrelated to API changes. It's basically stating that each `OctColor` pixel can be represented by 4 bits. This allows a raw buffer of 4 bit OctColor pixels to be readily converted into `ImageRaw`.
See [Raw Color Types Docs](https://docs.rs/embedded-graphics/0.7.1/embedded_graphics/pixelcolor/raw/index.html)

Comment (caemor) [reply src/color.rs]: :+1:

**Comment by caemor** [comment]
Is it ready to merge from your side?
From my side it looks good to go.

**Comment by crzysdrs** [comment]
Yep, all good.

**Comment by caemor** [comment]
Thanks for your work!

**Pull request merged**